[Build] Add windows building configurations (#169)
apache/pulsar-client-node: README.md

## How to install

> ### Install in windows

Reviewer: ```suggestion
### Install on windows
```

> You need to clone the Pulsar repo and build the Pulsar c++ client in windows, refer to https://pulsar.apache.org/docs/en/next/client-libraries-cpp/.

Reviewer: ```suggestion
1. [Start Pulsar](https://pulsar.apache.org/docs/en/next/standalone/). 

2. [Build the Pulsar C++ client on windows](https://pulsar.apache.org/docs/en/next/client-libraries-cpp/).
```

do you mean this?

> After building pulsar-client-cpp, you could open windows cmd tool, and using the pulsar-client-cpp path to set the variable `PULSAR_CPP_DIR`.

Reviewer: ```suggestion
3. Set the variable `PULSAR_CPP_DIR` with the `pulsar-client-cpp` path in a Windows command tool.
```

> Open windows cmd tool and set the variable `PULSAR_CPP_DIR`, for example.

Reviewer: ```suggestion

```

Reviewer: delete this since it is duplicated with the previous sentence?

```shell
set PULSAR_CPP_DIR=C:\pulsar\pulsar-client-cpp
```

> ### Please install pulsar-client in your project:

Reviewer: ```suggestion
### Install pulsar-client to your project
```

$ npm install
```

> > (Notice: If you build pulsar-client-node in windows,you need to set the variable `PULSAR_CPP_DIR` first, and run the command `npm install` in cmd tool)

Reviewer: ```suggestion
> **Note**
>
> If you build `pulsar-client-node on` windows, you need to set the variable `PULSAR_CPP_DIR` first and then install npm (run the command `npm install`) later in a Windows command-line tool.
```

do you mean this?
